### PR TITLE
chore: Update dependency requirements. Fix node version manager.

### PR DESCRIPTION
**Pull request by TeffenEllis**
This PR addresses some minor doc changes, and adds files that allow NVM and nodenv to automatically switch to our preferred version.

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+.node-version
```
Comment (TeffenEllis) [inline .nvmrc:1]: Included as a symlink to avoid version drifting between NVM and nodenv

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -29,6 +29,7 @@ Here is what is needed:
 
 - `node` v14.x
 - `git` v2.x or greater
+- [`git-lfs`](https://git-lfs.github.com)
```
Comment (TeffenEllis) [inline docs/CONTRIBUTING.md:32]: I encountered a somewhat mysterious error when building the project on a clean slate. Turns out we need this too!

File: .node-version [hunk 3]
```diff
@@ -0,0 +1 @@
+14
```
Comment (jawnsy) [inline .node-version:1]: I dunno anything about nvm, though I'm curious why these tools don't inspect package.json's engine info for this?

Seems like a reasonable change to me though!

**Review by jawnsy — APPROVED** [review]

File: .node-version (hunk 3 above)
Comment (jsjoeio) [reply .node-version:1]: Agreed, you would think it would be smarter!

**Review by jsjoeio — APPROVED** [review]
Awesome work! Thanks for updating all this 🎉

File: .node-version (hunk 3 above)
Comment (TeffenEllis) [reply .node-version:1]: Ahh, [open source](https://github.com/nvm-sh/nvm/issues/651) 😆

**Comment by jsjoeio** [comment]
Minor note: if your PR isn't related to an issue, add the Milestone on the right side so we associate this work with the next release.

**Pull request merged**